Sources/Tqdm/Tqdm.swift

```diff
@@ -257,6 +258,12 @@ public class Tqdm {
         }
     }
 
+    // Complete Progress
+    public func complete() {
+            self.n = self.total ?? 0
```

Comment:
Making `self.n` 0 seems wrong to me. Keeping it same in case of `self.total = nil` is a better approach in my opinion.

Other than that, `close()` call is not clearly documented. Can you add it to the documentation so that caller is informed? Something like `/// Complete and close progress bar` would be OK.

Reply:
OK Will make these changes.